tsdb/querier_bench_test.go

```diff
@@ -55,6 +55,8 @@ func BenchmarkQuerier(b *testing.B) {
 			addSeries(labels.FromStrings("i", strconv.Itoa(i)+postingsBenchSuffix, "n", "1_"+strconv.Itoa(n)+postingsBenchSuffix, "j", "bar"))
 			addSeries(labels.FromStrings("i", strconv.Itoa(i)+postingsBenchSuffix, "n", "2_"+strconv.Itoa(n)+postingsBenchSuffix, "j", "foo"))
 		}
+		require.NoError(b, app.Commit())
+		app = h.Appender(context.Background())
```

Comment:
Just out fo interest: In `createHeadForBenchmarkSelect` you added a commit every 1000 appends, here 100k*c appends are done. Is there a technical reason?
Otherwise might be handy to have a `func maybeCommit(app, i, b)` and just call that in the respective loops?

Reply:
No reason; I probably copied the 1000 from somewhere but can't find it now.
There are many different styles, some of which look like errors, e.g. [here](https://github.com/prometheus/prometheus/blob/5fdec3140188808bb9c2bf48e94b8533dbf75ac5/tsdb/querier_test.go#L3504-L3508).
I agree it should be consistent but probably am not going to fix that today.